dev-tools/clean-setup.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 # EMBArk - The firmware security scanning environment
 #
-# Copyright 2020-2021 Siemens Energy AG
-# Copyright 2020-2021 Siemens AG
+# Copyright 2020-2022 Siemens Energy AG
+# Copyright 2020-2022 Siemens AG
```

Comment:
I think SAG has not contributed to this area. If so, please remove ... also in the other headeres